### PR TITLE
Add tables page

### PR DESCRIPTION
**Pull request by niqjohnson**
Add the tables page to the Design Manual (see #153)
## Additions
- Tables page
- Screenshots of small-screen tables
## Changes
- Updated the table color variables in `cf-theme-overrides.less` to match the new table specs
- Since I had to run `./setup.sh` to pull in the latest cf-tables package, I think all the latest Capital Framework packages got pulled in, too, so there are lots of changes to the CSS and JS
## Testing
- Check out http://niqjohnson.github.io/design-manual/page-components/tables.html. Everything there should look good.
- Everything else at http://niqjohnson.github.io/design-manual/ should work as it does in production
## Review
- @Scotchester 
## Screenshots

tables

### CODE REVIEW

**Comment by jimmynotjim** [comment]
The pagination buttons look really janky. @Scotchester any ideas what's up with that?

File: page-components/tables.md [hunk 1]
```diff
@@ -4,6 +4,1217 @@ title:  Tables
 category: Page components
 ---
 
+- [Use cases](#use-cases)
+- [Behavior](#behavior)
+- [Content guidelines](#content-guidelines)
+- [Style](#style)
+- [Code](#code)
```
Comment (jimmynotjim) [inline page-components/tables.md]: Do you need a code heading? There's nothing after it, and we link to the code throughout the rest of the page.

File: page-components/tables.md [hunk 2]
```diff
@@ -4,6 +4,1217 @@ title:  Tables
...
+{: class="toc"}
+
+<div class="content-50 content-first">
+
+Tables divide information into distinct columns and rows to create an intersection “cell” where data is displayed.
```
Comment (jimmynotjim) [inline page-components/tables.md:15]: Does the Design Manual follow the 80 char line rule?

File: page-components/tables.md [hunk 3]
```diff
@@ -4,6 +4,1217 @@ title:  Tables
...
+                National Day of Civic Hacking 2015
+            </td>
+            <td data-label="Date">
+                JUL 29, 2015
+            </td>
+        </tr>
+    </tbody>
+</table>
+<nav class="post-pagination pagination" role="navigation" aria-label="Pagination" data-ajax-action="posts-paginated.html">
+    <a class="btn btn__super btn__disabled pagination_prev">
```
Comment (jimmynotjim) [inline page-components/tables.md:452]: Maybe the `btn__super` class is the issue with the janky pagination?

**Comment by schaferjh** [comment]
Left field question/confirmation: @niqjohnson will this new tables page wholesale replace the current tables page here: https://cfpb.github.io/design-manual/page-components/tables.html? (And hopefully keep the same URL so no broken links?)

**Comment by niqjohnson** [comment]
@schaferjh: Yep, once this page is merged it'll replace what's at https://cfpb.github.io/design-manual/page-components/tables.html, so no current links that point there should break.

File: page-components/tables.md (hunk 1 above)
Comment (niqjohnson) [inline page-components/tables.md]: Ah, thanks, forgot to remove that blank section. It's gone now.

File: page-components/tables.md (hunk 2 above)
Comment (niqjohnson) [inline page-components/tables.md:15]: We haven't stuck to an 80-character line limit in other Design Manual pages, so I kept long lines in this page, too.

File: page-components/tables.md (hunk 3 above)
Comment (niqjohnson) [inline page-components/tables.md:452]: Looks like `.btn__super` is in the pagination example at https://cfpb.github.io/capital-framework/components/cf-pagination/, too.

I think the issue might be squeezing the pagination element into a narrow column. Not sure there's a good solution for that.

**Comment by niqjohnson** [comment]
@Scotchester, @jimmynotjim: Any thoughts on whether the tables page should be merged now? I tried to track down the paginator issues noted above, but I think it's just a side effect of putting the pagination in a narrow column. I don't think it detracts too much from the rest of the page, though.

**Review by Scotchester — APPROVED** [review]
Yeah, I think this is fine to go. The pagination height jank is present on the docs page, as you point out, @niqjohnson, and is unrelated to the changes in this PR.

**Comment by jimmynotjim** [comment]
Ditto Scott's comment. 👍

**Pull request merged**